model/sanbox.rb

```diff
@@ -0,0 +1,13 @@
+class X
```

Comment:
It's really great to have a template to follow for each model. I really appreciate this.